lib/prop/limiter.rb

```diff
@@ -150,13 +154,18 @@ def _throttle(strategy, handle, key, cache_key, options)
         return [false, strategy.zero_counter] if disabled?
 
         if leaky_bucket_strategy?(strategy)
-          return Prop::LeakyBucketStrategy._throttle_leaky_bucket(handle, key, cache_key, options)
+          is_over_limit, bucket_info_hash = Prop::LeakyBucketStrategy._throttle_leaky_bucket(handle, key, cache_key, options)
+          bucket_counter = bucket_info_hash.fetch(:bucket)
```

Comment:
what do you think of this, @KJTsanaktsidis ? 
I kept the counter as it is for the other strategies, as a plain number. 

All the other info is put into options as you can see in the test.

Reply:
👍 this makes sense!